openxla-nvgpu/compiler/src/openxla/compiler/nvgpu/Transforms/CMakeLists.txt

```diff
@@ -17,13 +17,15 @@ iree_cc_library(
     "Passes.h"
     "Passes.h.inc"
   SRCS
-    "Sample.cpp"
-  DEPS
-    ::PassesIncGen
-    MLIRIR
+        "ConvertMHLOToCUDNN.cpp"
+        DEPS
+        ::PassesIncGen
+        MhloDialect
+        MLIRIR
     MLIRPass
-    openxla::compiler::nvgpu::defs
-  PUBLIC
+        openxla::compiler::nvgpu::defs
+        openxla::compiler::nvgpu::Dialect::CUDNN::IR::IR
```

Comment:
`IR::IR`? Isn't it ok to omit the second one?

Reply:
Done.